python/tk_multi_breakdown2/dialog.py

```diff
@@ -604,16 +603,11 @@ def save_state(self):
             self._filter_menu.docked,
         )
 
-        if six.PY2:
-            self._settings_manager.store(
-                self.SPLITTER_STATE, self._ui.details_splitter.saveState()
-            )
-        else:
-            # For Python 3, store the raw QByteArray object (cannot use the settings manager because it
-            # will convert QByteArray objects to str when storing).
-            self._raw_values_settings.setValue(
-                self.SPLITTER_STATE, self._ui.details_splitter.saveState()
-            )
+        # For Python 3, store the raw QByteArray object (cannot use the settings manager because it
+        # will convert QByteArray objects to str when storing).
```

Comment:
```suggestion
        # Store the raw QByteArray object (cannot use the settings manager because it
        # will convert QByteArray objects to str when storing).
```